src/components/dev-hub/podcast-player/player-contols.js

```diff
@@ -0,0 +1,107 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+import Button from '~components/dev-hub/button';
+import {
+    FastForwardIcon,
+    PauseIcon,
+    PlayIcon,
+    RewindIcon,
+} from '~components/dev-hub/icons/audio-player';
+
+import { screenSize, size } from '~components/dev-hub/theme';
+
+const buttonStyles = css`
+    margin-right: 12px;
+    padding: ${size.tiny};
+    &:active {
+        filter: brightness(60%);
+    }
+`;
+
+const mainButtonHover = css`
+    svg {
+        transform: scale(1.25);
+        circle {
+            stroke-width: 3;
+            r: 38;
```

Comment:
Is this working on all browsers (safari/firefox/chrome), as SVG 1.1 does not support customizing `r` via CSS (https://stackoverflow.com/questions/14255631/style-svg-circle-with-css). Do you see it making an impact everywhere? Probably is but want to be sure because my IDE was flagging this for a warning.

Reply:
it works across all browsers

Reply:
Awesome thanks